docs/sphinx/conf.py

```diff
@@ -44,7 +44,7 @@
 
 # Configuration for the edit_on_github extension
 edit_on_github_project = 'openmicroscopy/ome-documentation'
```

Comment:
:+1: for correcting the branch, can you also update the repository name since this documentation has been migrated down to `ome-model`